tools/accuracy_checker/accuracy_checker/adapters/segmentation.py

```diff
@@ -146,3 +146,15 @@ def _extract_predictions(self, outputs_list, meta):
             output_map[output_key] = output_data
 
         return output_map
+
+class OAR3DSegmentationAdapter(Adapter):
```

Comment:
what the difference from original segmentation adapter?

Reply:
Custom segmentation adapter removed